src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,104 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+import { findParent } from '../utils/ParentLink';
+import { SearchLink } from './SearchLink';
+
+export type Props = {
+  people: Person[],
+  selectedPerson: string,
+  sort: string,
+  order: string,
+};
+const searchColumnParams = ['Name', 'Sex', 'Born', 'Died'];
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  selectedPerson = '',
+  sort,
+  order,
+}) => {
+  const getSortParams = (column: string) => {
+    if (column === sort && !order) {
+      return { sort: column, order: 'desc' };
+    }
+
+    if (column === sort && order) {
```

Comment:
Change the order to 'asc' when clicking on the same column again instead of resetting both sort and order.